scripts/scil_summarize_bundles_similarity_measures.py

```diff
@@ -0,0 +1,327 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import argparse
+import copy
+import hashlib
+import itertools
+import json
+import multiprocessing
+import os
+import shutil
+
+from dipy.io.stateful_tractogram import Space, StatefulTractogram
+from dipy.io.streamline import load_tractogram, save_tractogram
+from dipy.io.utils import is_header_compatible, get_reference_info
+from dipy.segment.clustering import qbx_and_merge
+import nibabel as nib
+import numpy as np
+from numpy.random import RandomState
+
+from scilpy.io.utils import (add_overwrite_arg,
+                             add_reference,
+                             assert_inputs_exist,
+                             assert_outputs_exist,
+                             link_bundles_and_references)
+from scilpy.tractanalysis.reproducibility_measures \
+    import (compute_dice_voxel,
+            compute_bundle_adjacency_streamlines,
+            compute_bundle_adjacency_voxel,
+            compute_dice_streamlines,
+            get_endpoints_map)
+from scilpy.tractanalysis.streamlines_metrics import compute_tract_counts_map
+
+
+DESCRIPTION = """
+Compute pair-wise similarity measures of bundles.
+All tractograms must be trk files and headers must be identical.
+"""
+
+
+def _build_args_parser():
+    p = argparse.ArgumentParser(
+        description=DESCRIPTION,
+        formatter_class=argparse.RawTextHelpFormatter)
+    p.add_argument('in_bundles', nargs='+',
+                   help='Path of the input bundles')
+    p.add_argument('out_json',
+                   help='Path of the output json file')
+
+    p.add_argument('--same_tractogram', action='store_true',
+                   help='Streamlines-wise Dice coefficient will be computed \n'
+                        '[%(default)s]')
+    p.add_argument('--disable_streamline_distance', action='store_true',
+                   help='Will not compute the streamlines distance \n'
+                        '[%(default)s]')
+    p.add_argument('--single_compare',
+                   help='Compare inputs to this single file')
+    p.add_argument('--files_exist', action='store_false',
+                   help='Disable the verification of input files')
```

Comment:
I don't see the difference with overwrite

Reply:
It is actually to not run `assert_input_exist`. When the command is being generated automatically this is useful.

However I can remove it anyway since the code can be easily commented when, if, anyone is ever in that situation.